pkg/features/features.go

```diff
@@ -47,6 +48,13 @@ const (
 	// alpha: v0.1.6
 	PodPreset utilfeature.Feature = "PodPreset"
 
+	// CatalogRestrictions controls weather the ClusterServiceBroker will use
+	// the catalog restrictions when doing a re-list operation for catalog
```

Comment:
The re-list wording makes it sound like it doesn’t ap0y the first time, if set

Reply:
There is no list operation in our code, it is always a re-list. But I can change the comment to be list/re-list.